src/views/manage-patches-view/layout.test.tsx

```diff
@@ -0,0 +1,44 @@
+import * as crypto from "crypto";
+
+import React from "react";
+
+import { render, screen } from "@testing-library/react";
+
+import { ManagePatchesLayout } from "./layout";
+
+import * as auth from "@mtfh/common/lib/auth/auth";
+
+const testAssetId = crypto.randomBytes(20).toString("hex");
+
+beforeEach(() => {
+  jest.resetAllMocks();
+  jest.spyOn(auth, "isAuthorisedForGroups").mockReturnValue(true);
+
+  // server.use(
+  //   rest.get(
+  //     `/api/v1/patch/${mockAssetPatch.parentId}`,
+  //     (req, res, ctx) => res(ctx.status(200), ctx.json(mockAssetArea)),
+  //   ),
+  // );
```

Comment:
Can this removed?

Reply:
Removed in e7b3a558419a073330e7509bb566530403b685c9